activiti-cloud-starter-runtime-bundle/src/test/java/org/activiti/cloud/starter/tests/services/audit/AuditProducerIT.java

```diff
@@ -860,7 +861,43 @@ public void shouldProduceEventsDuringMultiInstanceSubProcessExecution() {
                     RuntimeEvent::getProcessInstanceId)
                     .contains(tuple(PROCESS_COMPLETED, startProcessEntity.getBody().getId()));
         });
+    }
+
+    @Test
+    public void shouldHaveAppVersionSetInBothEventsAndApplicationElementEntities() {
+
+        //when
+        ResponseEntity<CloudProcessInstance> startProcessEntity = processInstanceRestTemplate.startProcess(ProcessPayloadBuilder
+                                                                                                                   .start()
+                                                                                                                   .withProcessDefinitionKey(SIMPLE_PROCESS)
+                                                                                                                   .withProcessDefinitionId(processDefinitionIds.get(SIMPLE_PROCESS))
+                                                                                                                   .withName("my instance name")
+                                                                                                                   .withBusinessKey("my business key")
+                                                                                                                   .build());
+        //then
+        await().untilAsserted(() -> {
+            List<CloudRuntimeEvent<?, ?>> receivedEvents = streamHandler.getAllReceivedEvents();
+
+            assertThat(streamHandler.getReceivedHeaders()).containsKeys(ALL_REQUIRED_HEADERS);
+
+            assertThat(receivedEvents)
+                    .filteredOn(event -> event.getEntity().getClass().getSuperclass().equals(ApplicationElementImpl.class))
```

Comment:
`event.getEntity().getClass().getSuperclass().equals(ApplicationElementImpl.class)` can be extracted in a variable `isApplicationElement` and reused in the next statements.